package.json

```
@@ -23,7 +23,7 @@
   },
   "devDependencies": {
     "common-tags": "^1.8.2",
-    "eslint": "^9.4.0",
+    "eslint": "https://github.com/eslint/eslint.git#fix/update-peer-deps",
```

```suggestion
    "eslint": "^9.11.1",
```

v9.11.1 has been released.